apps/meteor/tests/e2e/utils/mocks/subscriptionMock.ts

```diff
@@ -0,0 +1,83 @@
+export const subscriptionMock = [
```

Comment:
hey what do you think about using Types, like IUser and IRoom, just to prevent us to create some wrong mock?

Reply:
fix